README.md

```diff
@@ -122,31 +123,48 @@ client.tasks.findByTag(tagId, { limit: 5 }).then(function(collection) {
 });
 ```
 
-There are a few useful functions that can help you deal with
-paginated requests.
+Additionally, `Collection` has a few useful methods that can make them
+more convenient to deal with.
 
-#### Page iteration
+#### Individual page iteration
 
-To get the next page of a collection, you do not have to manually construct the next request.
-The `Client.nextPage` method takes care of this for you:
+To get the next page of a collection, you do not have to manually construct
+the next request. The `nextPage` method takes care of this for you:
+
+```js
+client.tasks.findByTag(tagId).then(function(firstPage) {
+  console.log(firstPage.data);
+  collection.nextPage().then(function(secondPage) {
+    console.log(secondPage.data);
+  });
+});
+```
+
+#### Automatic page iteration
+
+To automatically fetch a bunch of results and have the client transparently
+request pages under the hood, use the `all` method.:
```

Comment:
`all` feels like a misleading name for fetching N items. I prefer `take` (which is what the Ruby client does, I believe inspired by Clojure's lazy sequences), but I think `fetch` or `items` would also work.